nuxeo-coldstorage/src/test/java/org/nuxeo/coldstorage/helpers/TestColdStorage.java

```diff
@@ -87,6 +91,18 @@
     @Inject
     protected CoreFeature coreFeature;
 
+    @After
+    public void tearDown() {
+        clear();
+    }
+
+    public void clear() {
+        // Remove the content from the Main bucket
+        ColdStorageS3TestHelper.getInstance().cleanMainBucket(AbstractTestColdStorageOperation.S3_GLACIER_UPLOAD_PROVIDER);
+        // Remove the content from the Glacier bucket
+        ColdStorageS3TestHelper.getInstance().cleanGlacierBucket(AbstractTestColdStorageOperation.S3_GLACIER_UPLOAD_PROVIDER);
+    }
```

Comment:
We can create a dedicated method on the `ColdStorageS3TestHelper` called `clear` which will call the needed ones with the right buckets names.